src/components/Form/MucCheckbox.vue

```diff
@@ -22,7 +26,11 @@
  */
 const modelValue = defineModel<boolean>({ default: false });
 
-defineProps<{
+const { label } = defineProps<{
+  /**
+   * id of checkbox
```

Comment:
```suggestion
   * ID of checkbox
   * required property, as this is needed to make the component work seamlessly with a screenreader.
```

I would like to have a statement like this in the props description to make sure the devs using this lib understand it. Could you add this to the other comments as well?